tests/ceph_ansible/test_ansible_upgrade.py

```diff
@@ -209,8 +209,7 @@ def compare_container_counts(pre_upgrade_counts, post_upgrade_counts, prev_insta
             if '-mon' in node:
                 count -= 1
         if pre_upgrade_counts[node] != count:
-            log.error("Mismatched container count post upgrade")
-            return 1
+            log.warning("Mismatched container count post upgrade")
```

Comment:
@sunilangadi2 I don't think we should remove the return code. The test will never fail here. what is the logical reason behind this. ?

Reply:
@rakeshgm In 4.2 new container name got added as ceph-crash this was the issue https://github.com/red-hat-storage/cephci/issues/233

Reply:
I updated issue. This looks like we are avoiding the count check. what will happen if a container does not actually come up. we will not even fail because the return code is removed.

Reply:
@rakeshgm Thanks, I just decreasing the count for ceph-crash and this solution also fails even any of the container didn't come up during upgrade as you suggested.